test/integration/webhook_test.go

```diff
@@ -668,11 +670,13 @@ func ensureWebhookService(name string) (func() error, error) {
 	return closer, nil
 }
 
-func waitForWebhookService(t *testing.T) {
-	require.Eventually(t, func() bool {
-		_, err := net.DialTimeout("tcp", fmt.Sprintf("%s:%d", testutils.AdmissionWebhookListenHost, testutils.AdmissionWebhookListenPort), 1*time.Second)
-		return err == nil
-	}, ingressWait, waitTick, "waiting for the admission service to be up")
+func waitForWebhookServiceConnective(configResourceName string) error {
+	svcName := fmt.Sprintf("webhook-%s", configResourceName)
+	svcPort := 443
+	ctx, cancel := context.WithTimeout(context.Background(), ingressWait)
+	defer cancel()
+
+	return networking.WaitForConnectionOnServicePort(ctx, env.Cluster().Client(), controllerNamespace, svcName, svcPort, time.Second)
```

Comment:
Just one second of wait time? Isn't that a little too short?

Reply:
this is the dial timeout, that means timeout after 1 second if no TCP package (`SYN+ACK` or `RST`) from the other side. Since the original `waitForWebhookService` function used this dial timeout, I used the same value here. If you think 1 second is too short, then what is your suggestions for the value here? I would use 10s, becuase this is the default timeout of API calls used for calling admission webhooks.

Reply:
It just struck me, the difference between 10s used in https://github.com/Kong/gateway-operator/pull/139/files and 1s used here. Are we using that different timeouts on webhook definitions here and in operator tests?

Reply:
changed to 10s.